content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Jellyfin 10.5"
+subtitle = "A step closer to the best experience with your media"
+author = "Fernando"
+githubusername = "ferferga"
+date = "2020-03-06"
+justify = "center"
++++
+
+Hello Jellyfin users out there! Welcome back to our blog!
+
+What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
```

Comment:
What would you think if I **told** you that 10.5 is probably the biggest release we've **ever** done? Fasten your seat belts and get ready because the numbers are big: